docs/EN/Getting-Started/FAQ.md

```diff
@@ -88,8 +88,20 @@ You can remove the pump while taking a shower or bath. For this short period of
 
 ## Sleeping
 
+### How can I loop during the night without mobile and WLAN radiation?
```

Comment:
I believe the more common term in English is WIFI.

Reply:
Thank you Adrian. Fixed.